webapp/src/main/java/life/qbic/views/register/UserRegistrationHandler.java

```diff
@@ -39,48 +39,53 @@ public void handle(UserRegistrationLayout registrationLayout) {
   }
 
   private void initFields() {
+    userRegistrationLayout.fullName.setPattern(".{1,}");
```

Comment:
I do not think this validation is correct. We want a non-empty full name. That requires at least one non-whitespace character.
```suggestion
    userRegistrationLayout.fullName.setPattern("\\S.*");
```
With the requirement: Your name must start with a non-whitespace character.
If we want to allow whitespace in front of the name (and strip it later .. although I do not know why we would do that) the expression would be
```suggestion
    userRegistrationLayout.fullName.setPattern("\\s*\\S.*");
```